target-app/Home/Views/SaveTargetViewController.swift

```diff
@@ -92,13 +92,26 @@ class SaveTargetViewController: UIViewController {
         case titleField:
             viewModel.setTitle(title: newValue)
         case topicsField:
+            //TODO: fix send ID value instead label string
             viewModel.setTopicId(topicId: newValue)
         default: break
         }
     }
+    
+    
+    // MARK: - BINDERS
+    private var cancellables = Set<AnyCancellable>()
+    
+    func setupBinders() {
+        viewModel.$topics
+            .sink{ [weak self] topics in
+                self?.topics = topics
+            }.store(in: &cancellables)
```

Comment:
For now, I need it to get the IDs easily before make the API call in `setTopicId` function. After refactor, perhaps is not needed anymore because I intent to send the topicID in the param.

```
  func setTopicId(topicId: String) {
      //TODO: fix receive ID instead label string
      if let id = topics.first(where: { $0.label == topicId } )?.id {
          self.topicId = id
      }
  }
```